make_in/Makefile.in

```diff
@@ -75,6 +75,11 @@ ifneq ($(ARCH),$(filter $(ARCH),ppc64 ppc64le))
   COMMON_CFLAGS += -march=nocona
 endif
 
+# The type char is unsigned by default on PPC -- we want signed
+ifeq ($(ARCH),$(filter $(ARCH),ppc64 ppc64le))
+  COMMON_CFLAGS += -fsigned-char
```

Comment:
fix by making the code where the `char` needs to be `signed`, and make that as a `signed char`

Reply:
i'm going to leave this as-is for now. if i have more time or if aerospike wants to fix each individual char, then we'll go that route.